.github/workflows/on-pr.yaml

```diff
@@ -100,6 +100,9 @@ jobs:
       - name: NPM Install
         run: npm install
 
+      - name: Start MongoDB Instance
+        uses: supercharge/mongodb-github-action@1.8.0
```

Comment:
You can use github action services to run mongodb automatically

Reply:
https://docs.github.com/en/actions/using-containerized-services/about-service-containers

Reply:
Oh cool! Thanks for the reference, updated to use the service approach